src/clarity-angular/emphasis/alert/alert.module.ts

```diff
@@ -21,6 +21,9 @@ import { ALERT_DIRECTIVES } from "./index";
     ],
     exports: [
         ALERT_DIRECTIVES
+    ],
+    schemas: [
```

Comment:
Weird. It actually shouldn't because you are already importing clrIconsModule too. Thats what datagrid and other components do too.